feature/qos/otg_tests/egress_traffic_classification_and_rewrite_test/README.md

```diff
@@ -0,0 +1,205 @@
+
+# DP-1.17: Egress traffic DSCP rewrite
+## Summary
+
+This test validates egress traffic scheduling and packet remarking (rewrite) on a DUT. IP payload encapsulated as IPoGRE,
+IPoMPLSoGRE,IPoGUE or IPoMPLSoGUE would be decapsulated by DUT as per tunnel termination configuration and forwarded as an 
+IP payload via the egress interface. IP payload should be scheduled via egress interface according to DSCP markings prior to rewrite.
+DSCP values of IP payload encapsulated as IPoGRE, IPoMPLSoGRE, IPoGUE or IPoMPLSoGUE should have payload DSCP values re-written 
+per egress QOS re-write policy applied to the egress interface. Egress QOS policy should support match conditions on payload 
+IP header fields, specifically DSCP values. Based on the match condition, DSCP must be set to a new value. DUT configuration 
+will be evaluated against OpenConfig QOS model, and traffic flows analyzed to ensure proper scheduling, re-marking, and forwarding.
+
+## Testbed type
+
+* [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+* DUT has an ingress port and 1 egress port.
+
+| | [ ATE Port 1 ] ----- | DUT | ----- [ ATE Port 2 ] | |
+
+
+* Configure DUT's ingress and egress interfaces.
+
+
+#### Configuration
```

Comment:
It seems like the tests in this README depend on there being some specific configuration as to what queues exist on the egress interface. Is my understanding correct? If so, can we specify what this mapping is.